posts/aws-callrail-ippon-blog.md

```diff
@@ -0,0 +1,38 @@
+---
+authors:
+- Lucas Ward, Wanchen Zhao
+tags:
```

Comment:
I would consider adding more tags, such as "Migration" and "DB Migration"

Reply:
Done and Done, thank you for your service